src/networking.c

```diff
@@ -3751,6 +3759,8 @@ int handleClientsWithPendingReadsUsingThreads(void) {
 
 /* Returns true if client memory limit was reached */
 int clientEvictionCheckLimit() {
+    serverLog(LL_WARNING, "Checking limits: clients memory: %zu", server.stat_clients_type_memory[CLIENT_TYPE_NORMAL] +
```

Comment:
i suppose you left this debug print by mistake? or temporary to be trimmed later?

Reply:
Yes, my mistake. Fixed it.